CompetitionTemplate/src/competition/subsystems/arm/arm_commands/DisableSafeArmOperationCommand.java

```diff
@@ -0,0 +1,26 @@
+package competition.subsystems.arm.arm_commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.arm.ArmSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class DisableSafeArmOperationCommand extends BaseCommand{
+    ArmSubsystem armSubsystem;
+    
+    @Inject
+    public DisableSafeArmOperationCommand(ArmSubsystem armSubsystem){
+        this.armSubsystem = armSubsystem;
+        this.requires(this.armSubsystem);
```

Comment:
I don't think you want to require the subsystem or it'll prevent interrupt other arm commands